app/Http/Controllers/Bible/BiblesController.php

```diff
@@ -116,15 +116,14 @@ public function index()
             }
         }
 
-        $access_control = $this->accessControl($this->key);
```

Comment:
this looks like code which has already been merged. Is there a way to clean this up so I only look at relevant code?